macros/graph/plots.pl

```diff
@@ -106,6 +106,13 @@ =head2 PLOT FUNCTIONS
     # Add a parametric circle of radius 5 to the plot.
     $plot->add_function(['5cos(t)', '5sin(t)'], 't', 0, 2*pi);
 
+Polar functions can be graphed using the C<< polar => 1 >> option with a single variable
```

Comment:
Up on line 31, you have an example with `title`, which I think the functionality has been removed.